cvat/apps/engine/static/engine/js/player.js

```diff
@@ -123,6 +123,7 @@ class FrameProvider extends Listener {
 
 const MAX_PLAYER_SCALE = 10;
 const MIN_PLAYER_SCALE = 0.1;
+const PLAYER_FRAME_OFFSET = 1000;
```

Comment:
Let's keep drawing canvas with fixed size (image_w + 2*offset, image_h + 2*offset). Player fit should focus on the image itself as previously

Reply:
Done